resources/pgp-keys-map.list

```diff
@@ -216,6 +216,8 @@ geronimo-spec                   = noSig
 
 info.picocli                    = 0x8756C4F765C9AC3CB6B85D62379CE192D401AB61
 
+informa:informa:0.6.0           = noSig
```

Comment:
There is only this version
https://repo1.maven.org/maven2/informa/informa/
so specific versions range here is not need now

Reply:
I have rebased and removed the version range.